modules/storage-persistent-storage-nfs-group-ids.adoc

```diff
@@ -11,7 +11,7 @@ change permissions on the NFS export, is to use supplemental groups.
 Supplemental groups in {product-title} are used for shared storage, of
 which NFS is an example. In contrast block storage, such as
```

Comment:
_While you are here:_
s/In contrast block storage, such as iSCSI, use the/In contrast, block storage such as iSCSI uses the/
Thanks 😀